### PR TITLE
DOCS-4919 sudo and root info

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/config-workflow.adoc [hunk 1]
```diff
@@ -18,3 +18,7 @@ endif::[]
     * Register Mule runtimes
 
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
+
+[NOTE]
+PCE containers and processes must run as root for port binding and system and kernel configuration. Containers running as root
```
Comment (federicoboerr) [inline modules/ROOT/pages/config-workflow.adoc]: Good start. Not all of the containers run as root. I'd say "A few docker containers must run as ..."

File: modules/ROOT/pages/config-workflow.adoc [hunk 2]
```diff
@@ -18,3 +18,7 @@ endif::[]
...
+are not exposed outside the cluster. All pods that are exposed outside the cluster run as non root.
```
Comment (federicoboerr) [inline modules/ROOT/pages/config-workflow.adoc]: All ~pods~ `containers` that are exposed outside ...

**Review by federicoboerr — COMMENTED** [review]
small comments

File: modules/ROOT/pages/config-workflow.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/config-workflow.adoc]: Got it - thank you!

File: modules/ROOT/pages/config-workflow.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/config-workflow.adoc]: Got it - thanks!

File: modules/ROOT/pages/config-workflow.adoc [hunk 3]
```diff
@@ -20,5 +20,5 @@ endif::[]
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
 
 [NOTE]
-PCE containers and processes must run as root for port binding and system and kernel configuration. Containers running as root
-are not exposed outside the cluster. All pods that are exposed outside the cluster run as non root.
+Some Docker containers must run as root for port binding and system and kernel configuration. Containers running as root
```
Comment (federicoboerr) [inline modules/ROOT/pages/config-workflow.adoc]: @paigehargrave `port binding` and `system and kernel configuration` can ring a security bell in our customers.
I'd just mention the fact that it's how it is now, without an explanation on why. And also add that we are working on moving away from it.

Comment (paigehargrave) [reply modules/ROOT/pages/config-workflow.adoc]: Updated this - thank you!

File: modules/ROOT/pages/config-workflow.adoc [hunk 4]
```diff
@@ -18,3 +18,7 @@ endif::[]
     * Register Mule runtimes
 
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
+
+[NOTE]
+At the time of this writing, some Docker containers must run as root. Current work in progress removes the need for these containers to run as root. Containers running as root
```
Comment (federicoboerr) [inline modules/ROOT/pages/config-workflow.adoc]: processes running as root doesn't expose functionality outside the cluster. All processes serving traffic to outside the cluster run as non-root, (reducing the attack surface)?

Comment (paigehargrave) [reply modules/ROOT/pages/config-workflow.adoc]: @federicoboerr PTAL at latest updates - thank you!

**Review by federicoboerr — APPROVED** [review]
LGTM

File: modules/ROOT/pages/config-workflow.adoc [hunk 5]
```diff
@@ -18,3 +18,8 @@ endif::[]
     * Register Mule runtimes
```
Comment (kbostrom) [inline modules/ROOT/pages/config-workflow.adoc:18]: Register Mule Runtime Engines?

File: modules/ROOT/pages/config-workflow.adoc [hunk 6]
```diff
@@ -18,3 +18,8 @@ endif::[]
...
 
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
```
Comment (kbostrom) [inline modules/ROOT/pages/config-workflow.adoc]: comma before second "and":

"...register APIs, and use"

Comment (kbostrom) [reply modules/ROOT/pages/config-workflow.adoc]: ...and use API Exchange on...
(no "the")

File: modules/ROOT/pages/config-workflow.adoc [hunk 7]
```diff
@@ -18,3 +18,8 @@ endif::[]
...
+
+[NOTE]
+At the time of this writing, some Docker containers must run as root. Current work in progress removes the need for 
+these containers to run as root. Processes running as root do not expose functionality outside the cluster. 
+Processes that serve external traffic run as non root to maximize security.
```
Comment (kbostrom) [inline modules/ROOT/pages/config-workflow.adoc]: Hmm...not sure we are supposed to document future functionality. Can we weasel around this?

Currently, some Docker containers must run as root.

Then we would omit the second sentence.

Comment (kbostrom) [reply modules/ROOT/pages/config-workflow.adoc]: non root > Processes that serve external traffic do not run as root, to maximize security.

Doesn't that contradict the previous statement?

Or "non root" > nonroot, non-root (if you think one word is confusing)

File: modules/ROOT/pages/install-workflow.adoc [hunk 8]
```diff
@@ -14,9 +14,13 @@ The installer is interactive and walks you through a set of wizard pages that en
...
 
 [NOTE]
-The installation package includes everything required for Anypoint Private Cloud Edition installation, 
+====
+
+- The installation package includes everything required for Anypoint Private Cloud Edition installation, 
 including specific versions of additional software, such as Docker and Kubernetes. MuleSoft provides updates as 
-needed for the additional software with Anypoint Private Cloud Edition patches.  
+needed for the additional software with Anypoint Private Cloud Edition patches. 
+- You must run the installer as root in order to access the necessary privileged ports.
```
Comment (kbostrom) [inline modules/ROOT/pages/install-workflow.adoc]: in order to > to

**Review by kbostrom — APPROVED** [review]

File: modules/ROOT/pages/config-workflow.adoc (hunk 6 above)
Comment (paigehargrave) [reply modules/ROOT/pages/config-workflow.adoc]: Got it - thank you!

File: modules/ROOT/pages/config-workflow.adoc (hunk 7 above)
Comment (paigehargrave) [reply modules/ROOT/pages/config-workflow.adoc]: Updated - thank you! The future functionality statement was originally put in b/c this has been (and currently is) a concern for customers. Put in the statement to tell them it's being worked on, but might as well just document it in release notes when it is truly done.

**Pull request merged**